### DISCUSSION TITLE
Naming, Branding and Logo

### DISCUSSION THREAD

**Discussion by antfu** [windicss/windicss#22] [Ideas]
I feel there is a huge potential for this project in the near future and it will be compared with Tailwind quite frequently. I think it's nice to have a good naming conversion and some logo and branding stuff now.

## Naming Conversion

I saw you are mention this project mostly as `windicss` while the official name for tailwind is `Tailwind CSS`. Would love to know what' the naming conversion you'd prefer:

- windicss
- Windi CSS
- WindiCSS
- WindiCss

I personally would pick `Windi CSS` as it matches with tailwind's and in many context we can have `Windi vs Tailwind` or `Tailwind CSS and Windi CSS`. The current lowercase one feels less like a tool's name - `windicss vs Tailwind CSS`.

## Brand Color

Clearly, Tailwind uses teal(#06b6d4) as its brand color. Guess we should also pick up a color for Windi so it can be better comparable.

Do you have any color in mind?

## Logo

Wondering what's the meaning/inspiration for the name `windi`? If you'd like to, I am happy to draft some logo designs for it.

Would love to hear what you think about this, thanks!

**Comment by antfu** [comment]
BTW, I have just secured the npm namespace [@windicss](https://www.npmjs.com/org/windicss) and GitHub org [windicss](https://github.com/windicss) for the potential growth of this project. Can you have your npm id so I can give your permission? (just in case you might need them in the future, not trying to push anything)

**Comment by voorjaar** [comment]
It looks weird if the name are separated. Tailwind is also tailiwindcss most of the time. Look at their [logo](https://camo.githubusercontent.com/53b9876cd8e38928387c6824043b0e2772b15b1bfdb7f42d0864216abbf3dfe8/68747470733a2f2f7265666163746f72696e6775692e6e7963332e63646e2e6469676974616c6f6365616e7370616365732e636f6d2f7461696c77696e642d6c6f676f2e737667)

About the name, you know that there is a movie called Billions and the heroine is called Wendy, so I took a windi, which is also very close to tailwind.

Colors that I prefer:

#339AF0

#5335EF

#7F9CF5

Since you are a designer, you can choose the colors according to the graphics.

**Comment by antfu** [comment]
> npm id

Sent. Let me know when you accepted it so I can update the role.

> name

Sounds reasonable. Are you ok with calling it Windi for most of the time (refer to Tailwind)? Imagine this page https://tailwindcss.com/ but replace all the `Tailwind` to `Windi`, would the usages good to you?

> logo

Sure, I will give it a try bit might take some time. I think we should also welcome proposals from the community ;)

**Reply by voorjaar** [reply]
Already joined. Windi is fine, I always feel that adding CSS at the end is weird, probably because it doesn't conform to any variable naming rules. And take your time, you have helped a lot.

**Comment by nico-bachner** [comment]
How about a domain name? The Netlify subdomain is fine for the time being, but when the project takes off, it would probably be a good idea to look into a custom domain (like Tailwind has `tailwindcss.com`).

There are two main things to keep in mind here.

First, we should decide on the name itself. I assume it would most likely be either `windicss` or `windi` (maybe also `windi-css`) if it is to follow the naming conventions established above. @voorjaar and @antfu, I'd love to hear both your opinions on the subject.

Second, the finance part. We should probably contact some domain name registrars to see if any are interested in providing one for free. In my opinion, a `.dev` domain would be the most fitting for the project, but again, I think it would be best to discuss this together.
If that fails, I guess we could set up GitHub sponsors and finance it through there. 

Look forward to hearing everyone's opinions on the matter.

**Reply by voorjaar** [reply]
Yes, I also think there should be a domain name. If it is abbreviated, windicss is better. Like windicss.com or windicss.dev, I just searched, and neither of these domain names has been registered yet. Regarding the financial aspect, you have explained very comprehensively. But I don't know much about this aspect, and may have to wait for Anthony to come and discuss it together.

**Reply by antfu** [reply]
https://windicss.org/ 😎
